packages/utils/telemetry-utils/src/error.ts

```diff
@@ -184,6 +184,8 @@ export class DataProcessingError extends LoggingError implements IErrorBase, IFl
  * Extracts specific properties from the provided message that we know are safe to log.
  *
  * @param messageLike - Message to include info about via telemetry props.
+ *
+ * @public
```

Comment:
Nope

Reply:
+1. Confirmed no uses in bohemia nor loopsdk.